### ISSUE TITLE
Template is changed by label-studio automatically

### ISSUE THREAD

**Issue by thesby** [heartexlabs/label-studio#691]
I use the template:
```HTML
<View>
  <Pairwise name="pw" toName="img-1,img-2"/>
    <View style="padding:5px;">
    <Image name="img-1" value="$image1"/>
  </View>
  <View style="padding:5px;">
    <Image name="img-2" value="$image2"/>
  </View>
</View>
```
After saving and open settings again, the template becomes:
```HTML
<View>
  <Pairwise name="pw" toName="img-1,img-2"/>
  <View style="padding:5px;">
    <Image name="img-1" value="$image2"/>
  </View>
  <View style="padding:5px;">
    <Image name="img-2" value="$image1"/>
  </View>
</View>
```

More importantly, the image pair cannot be rendered correctly: two images shows the same image, the $image1.

If I change the template as :
```HTML
<View>
  <Pairwise name="pw" toName="img-1,img-2"/>
  <View style="padding:5px;">
    <Image name="img-1" value="$image1"/>
  </View>
  <View style="padding:5px;">
    <Image name="img-2" value="$image1"/>
  </View>
</View>
```
then two different images appears, but the first image becomes $image2, and the second image becomes $image1

**Comment by niklub** [comment]
Hi, @thesby ! Can you please check what version you'd used? Since the similar issue was supposed to be fixed starting from `label-studio==1.0.0.post2`

**Comment by thesby** [comment]
I have updated as `label-studio==1.0.0.post2`, but the problem is not fixed.

**Comment by thesby** [comment]
My template is always changed by label-studio now after I save it. Any suggestion? @niklub

**Comment by thesby** [comment]
This problem may be about the prefix of data field. I find that the same prefix of data field always be changed.

**Comment by niklub** [comment]
Hi, @thesby ,
Can you please show what kind of import format do you use for that? Just tried to unsuccessfully reproduce this error in [1.0.1 release branch](https://github.com/heartexlabs/label-studio/pull/681), importing csv file with 2 columns `image1` and `image2` and it works fine.

**Comment by thesby** [comment]
Hi, I made a json format data, and uploaded to server. Then I imported the data from url of the file

**Comment by niklub** [comment]
Hey, this problem is supposed to be fixed following `label-studio>=1.0.1`. Please feel free to reopen the issue in case it happens again.
